### ISSUE TITLE
Cannot read property 'tapAsync' of undefined

### ISSUE THREAD

**Issue by tannerhallman** [getsentry/sentry-webpack-plugin#163]
ISSUE:

Looks like my versions of `sentry-webpack-plugin` and webpack (?) don't play nicely together. `sentry-webpack-plugin` looks like it's assuming a webpack callback hook `afterEmit` exists, when it does not.

`package.json`
```
deps -- 
"webpack": "^4.34.0",
devDeps -- 
"@sentry/webpack-plugin": "^1.9.3",
```

`webpack.config.js`

```
...
 plugins: [
      new SentryWebpackPlugin({
        include: '.',
        ignoreFile: '.sentrycliignore',
        ignore: ['node_modules', 'webpack.config.prod.js'],
        configFile: 'sentry.properties',
      }),
    ],
...
```

Suspected error in `sentry-webpack-plugin`
`index.js`
<img width="767" alt="index js — coworks (Workspace) 2020-01-30 13-18-51(1)" src="https://user-images.githubusercontent.com/10327491/73478143-85924380-4363-11ea-88c4-c1ac30f780ed.png">

When printing `console.log(compiler.hooks)`, I get (don't see afterEmit):
``` 
{ resolveStep:
   SyncHook {
     _args: [ 'hook', 'request' ],
     taps: [],
     interceptors: [],
     call: [Function: lazyCompileHook],
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'resolveStep' },
  noResolve:
   SyncHook {
     _args: [ 'request', 'error' ],
     taps: [],
     interceptors: [],
     call: [Function: lazyCompileHook],
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'noResolve' },
  resolve:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'resolve' },
  result:
   AsyncSeriesHook {
     _args: [ 'result', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined },
  parsedResolve:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'parsedResolve' },
  describedResolve:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'describedResolve' },
  rawModule:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'rawModule' },
  module:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'module' },
  relative:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'relative' },
  describedRelative:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'describedRelative' },
  directory:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'directory' },
  existingDirectory:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'existingDirectory' },
  undescribedRawFile:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'undescribedRawFile' },
  rawFile:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'rawFile' },
  file:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [ [Object] ],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'file' },
  existingFile:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'existingFile' },
  resolved:
   AsyncSeriesBailHook {
     _args: [ 'request', 'resolveContext' ],
     taps: [],
     interceptors: [],
     call: undefined,
     promise: [Function: lazyCompileHook],
     callAsync: [Function: lazyCompileHook],
     _x: undefined,
     name: 'resolved' } }
```

**Comment by tannerhallman** [comment]
https://github.com/getsentry/sentry-webpack-plugin/pull/165

**Comment by kamilogorek** [comment]
Fixed in #165
